services/server/src/server/apiv1/verification/private/stateless/customReplaceMethods.ts

```diff
@@ -0,0 +1,164 @@
+        databaseColumns.onchainCreationCode.bytecode_hash_keccak,
+      ],
+    );
+
+    if (creationCodeInsertResult.rows.length === 0) {
+      // Code already exists, get the hash
+      const existingCodeResult = await poolClient.query(
+        `SELECT code_hash FROM code WHERE code_hash = digest($1::bytea, 'sha256')`,
+        [databaseColumns.onchainCreationCode.bytecode],
+      );
+      newCreationCodeHash = existingCodeResult.rows[0].code_hash;
+    } else {
+      newCreationCodeHash = creationCodeInsertResult.rows[0].code_hash;
+    }
+  }
+
+  // Get current contract's runtime code hash
+  const currentContractQuery = `
+    SELECT runtime_code_hash FROM contracts WHERE id = $1
+  `;
+  const currentContractResult = await poolClient.query(currentContractQuery, [
+    existingVerifiedContract.contract_id,
+  ]);
+  const runtimeCodeHash = currentContractResult.rows[0].runtime_code_hash;
+
+  // Insert new contract with new creation code and existing runtime code
+  const newContractInsertResult = await poolClient.query(
+    `INSERT INTO contracts (creation_code_hash, runtime_code_hash) 
+     VALUES ($1, $2) 
+     ON CONFLICT (creation_code_hash, runtime_code_hash) DO NOTHING 
+     RETURNING id`,
+    [newCreationCodeHash, runtimeCodeHash],
+  );
+
+  let newContractId: string;
+  if (newContractInsertResult.rows.length === 0) {
+    // Contract already exists, get the id
+    const existingContractResult = await poolClient.query(
+      `SELECT id FROM contracts WHERE creation_code_hash = $1 AND runtime_code_hash = $2`,
+      [newCreationCodeHash, runtimeCodeHash],
+    );
+    newContractId = existingContractResult.rows[0].id;
+  } else {
+    newContractId = newContractInsertResult.rows[0].id;
+  }
+
+  // Update contract deployment with new creation fields and new contract_id
+  await poolClient.query(
+    `UPDATE contract_deployments 
+     SET 
+       transaction_hash = $2,
+       block_number = $3,
+       transaction_index = $4,
+       deployer = $5,
+       contract_id = $6
+     WHERE id = $1`,
+    [
+      deploymentId,
+      databaseColumns.contractDeployment.transaction_hash,
+      databaseColumns.contractDeployment.block_number,
+      databaseColumns.contractDeployment.transaction_index,
+      databaseColumns.contractDeployment.deployer,
+      newContractId,
+    ],
+  );
+
+  // Update verified_contracts with creation match data
+  await poolClient.query(
+    `UPDATE verified_contracts 
+     SET 
+       creation_match = $2,
+       creation_values = $3,
+       creation_transformations = $4,
+       creation_metadata_match = $5
+     WHERE id = $1`,
+    [
+      existingVerifiedContract.id,
+      databaseColumns.verifiedContract.creation_match,
+      databaseColumns.verifiedContract.creation_values,
+      databaseColumns.verifiedContract.creation_transformations
+        ? JSON.stringify(
+            databaseColumns.verifiedContract.creation_transformations,
+          )
```

Comment:
vscode shows me an eslint error here. I just found that our CI doesn't run the lint check because it is configured wrongly... Will open a pr to fix

Reply:
My vscode doesn't

Reply:
Do you have the eslint extension installed?

Reply:
lol I forgot to install it after I moved back to vscode :)